### PR TITLE
Start robots and HQ in the center.

### LINKED ISSUE #10: Make all of the robots start from the HQ

### PR DESCRIPTION
**Pull request by sjager**
Resolves #10

### CODE REVIEW

**Comment by sjager** [comment]
This branch is ready to be merged.

File: src/resourceGathering/ResourceGatheringBuilder.java [hunk 1]
```diff
@@ -31,7 +32,7 @@ public Context build(Context<Object> context) {
 				ContinuousSpaceFactoryFinder.createContinuousSpaceFactory(null);
 		ContinuousSpace<Object> space =
 				spaceFactory.createContinuousSpace("space", context,
-						new RandomCartesianAdder<Object>(),
+						new SimpleCartesianAdder<Object>(),
```
Comment (jimmyhlee94) [inline src/resourceGathering/ResourceGatheringBuilder.java]: So was there actually any benefit to changing this into a simple cartesian adder?

File: src/resourceGathering/ResourceGatheringBuilder.java [hunk 2]
```diff
@@ -50,13 +51,19 @@ public Context build(Context<Object> context) {
...
+		
 		HQ = new Headquarters(space, grid);
 		context.add(HQ);
+		space.moveTo(HQ, space.getDimensions().getHeight()/2, space.getDimensions().getWidth()/2);
 		
 		for (int i = 0; i < robotCount; i++) {
-			context.add(new Robot(space, grid, HQ, maxFuelLevel, fuelRate, maxSensorRange, maxCommunicationRange, i));
+			final Robot robot = new Robot(space, grid, HQ, maxFuelLevel, fuelRate, maxSensorRange, maxCommunicationRange, i);
+			context.add(robot);
+			space.moveTo(robot, space.getLocation(HQ).getX(), space.getLocation(HQ).getY());
```
Comment (jimmyhlee94) [inline src/resourceGathering/ResourceGatheringBuilder.java:62]: because it looks like you're manually moving all the robots after adding them anyways

Comment (sjager) [inline src/resourceGathering/ResourceGatheringBuilder.java:62]: yeah, thats what you have to do regardless, afaik

**Pull request merged**